.github/workflows/run_shark_tank.yml

```diff
@@ -4,6 +4,7 @@ on:
   schedule:
     - cron: '0 13 * * *'
   workflow_dispatch:
+  pull_request:
```

Comment:
Maybe when you comment this out, also add a comment about uncommenting it. (wow that sentence was a trip)